javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,163 @@
+}
+
+getFullname("Benjamin", "Hughes", true, false); -> would return: Lady Benjamin Hughes as getFullname's value.
+
+/*================================================================*/
+
+
+
+/* TASK 2 - Event application */
+
+function getEventWeekday(daysLeftTillEvent) {
+    const daysOfWeek = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturay"];
+    const currentDate = new Date();
+    const result = (currentDate.getDay() + daysLeftTillEvent) % 7;
+    console.log("%cThe event will take place on " + `${daysOfWeek[result]}` + ".", "color: white; background-color: black; font-size: 25px");
+}
+
+getEventWeekday(8);
+
+/*================================================================*/
+
+
+/* TASK 3 - Weather wear */
+
+const clothing = {
+    hot: ["Swimwear", "Flip-Flops", "Sunglasses"],
+    cold: ["Boots", "Jacket", "Scarf"]
+};
+
+function suggestClothingBasedOnWeather(currentTemperature) {
+    if (currentTemperature <= 20) {
+        console.log("%cThe weather is COLD, you should be wearing: " + clothing.cold.join(" ") + ".", "color: white; background-color: black; font-size: 25px");
+    } else {
+        console.log("%cThe weather is HOT, you should be wearing: " + clothing.hot.join(" ") + ".", "color: white; background-color: black; font-size: 25px");
+    }
+}
+
+suggestClothingBasedOnWeather(22);
+suggestClothingBasedOnWeather(12);
+
+
+/*================================================================*/
+
+
+/* TASK 4 - Student manager */
+
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+    if (!studentName) return; /* If no student name is given return */
+    if (studentName == "queen") { /* If student is queen add it to array */
+        class07Students.push(studentName);
+    } else {
+        if (class07Students.length >= 6) { /* If there are more than 6 students log msg */
+            console.log("%cCannot add more than 6 students to class 07.", "color: white; background-color: black; font-size: 25px");
+        }
+        if (class07Students.includes(studentName)) { /* If  student is already in the array log msg*/
+            console.log("%cStudent: " + studentName + " is already in the class", "color: white; background-color: black; font-size: 25px");
+        } else {
+            class07Students.push(studentName); /* add student to array */
+        }
+    }
+}
+
+function getNumberOfStudents() {
+    console.log("%cThere are: " + class07Students.length + " students in class07. %c", "color: white; background-color: black; font-size: 30px");
+}
+
+/* TEST CASE */
+addStudentToClass("First Student");
+addStudentToClass("Second Student");
+addStudentToClass("Third Student");
+addStudentToClass("Fourth Student");
+addStudentToClass("Fifth Student");
+addStudentToClass("Sixth Student");
+addStudentToClass("First Student");
+addStudentToClass("queen");
+
```

Comment:
Always try to give appropriate names even it's just a practice. this will help you in long run.